mlserver/settings.py

```diff
@@ -190,6 +191,9 @@ class Config:
     grpc_max_message_length: Optional[int] = None
     """Maximum length (i.e. size) of gRPC payloads."""
 
+    grpc_response_latency_histogram : bool = False
```

Comment:
Would there be any caveat to removing this setting and always keeping it enabled? It seems reasonable (and aligns it with its REST counterpart - where I think the latency histogram is also always enabled).